_includes/accessibility-tests/a11y-note.html

```diff
@@ -1,4 +1,4 @@
-{% assign component_name = page.title | downcase %}
+{% assign component_name = page.parent | default: page.title | downcase %}
```

Comment:
> [!note]
> This update allows the component to pull the "parent" name for the component group. In this example, instead of the CTA reading "Use default button group accessibility tests" or  "Use segmented button group accessibility tests", it reads "Use button group accessibility tests".

We should confirm that this does not cause regressions on other a11y-notes already on the site.

Reply:
Confirming that the pages that use the `parent` frontmatter is have no regressions with this change.

- [x]  Button group
- [x]  Header
- [x]  Language selector
  - Doesn’t have the accessibility tests page yet
- [x]  Table